### PR TITLE
feat: onboarding page styling, global colors

### PR DESCRIPTION
**Pull request by me-liu**
[//]: # "Feel free to customize this template to your project's vibes"

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Added gray global colors
- Onboarding page styling
- Styling for different device sizes (attempted)

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 
Screenshot 2024-10-09 at 10 59 11 PM

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- Review all at once: only styling commentary is really needed

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- There's an odd white border around the screen despite min height and width settings...
- Currently uses default Inter font

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @celinechoiii

### CODE REVIEW

File: app/onboarding/page.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <Checkbox type="checkbox" />
+            <UpdateText>
+              I want to get updated when there's an event that matches my
+              interest
+            </UpdateText>
+          </UpdateContainer>
+        </Container>
+        <ButtonContainer>
+          <ContinueButton>
+            <ContinueText>Continue </ContinueText>
```
Comment (celinechoiii) [inline app/onboarding/page.tsx]: Delete the white space after Continue

File: app/onboarding/styles.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+`;
+
+export const Checkbox = styled.input`
+  width: 25px;
+  height: 25px;
+  border-style: solid;
+`;
+
+export const UpdateText = styled.text`
+  text-align: justify;
```
Comment (celinechoiii) [inline app/onboarding/styles.ts]: set `text-align` to left

File: app/onboarding/styles.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  margin-left: 10px;
+`;
+
+export const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  height: 80%;
+`;
+
+export const ContinueButton = styled.button`
```
Comment (celinechoiii) [inline app/onboarding/styles.ts:102]: currently, the centering of the "continue" text it slightly off. i got the styling code from figma dev mode, so try this:

```
export const ContinueButton = styled.button`
  width: 5.5rem;
  height: 2.25rem;
  background-color: ${COLORS.gray11};
  border-color: ${COLORS.gray11};
  border-style: solid;
  border-radius: 8px;
  display: inline-flex;
  padding: 8px 16px;
  justify-content: center;
  align-items: center;
  gap: 10px;
`;
```

File: app/onboarding/styles.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  width: 20%;
+  height: 7%;
+  background-color: ${COLORS.gray11};
+  border-color: ${COLORS.gray11};
+  border-style: solid;
+  border-radius: 10px;
+`;
+
+export const ContinueText = styled.p`
```
Comment (celinechoiii) [inline app/onboarding/styles.ts:116]: get rid of `margin-top` field and adjust `font-size` to 14px

File: app/onboarding/styles.ts [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  height: 53%;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  margin: 25px 0px;
+  justify-content: space-between;
+  background-color: white;
+  border-radius: 8px;
+  padding: 13%;
+  box-shadow: 5px 5px 5px ${COLORS.gray8};
```
Comment (celinechoiii) [inline app/onboarding/styles.ts]: delete the `box-shadow` field (updated figma design)

File: app/onboarding/page.tsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+          <Input />
+          <text>
+            {' '}
+            Phone Number <RedAsterisk>*</RedAsterisk>{' '}
+          </text>
+          <Input />
+          <UpdateContainer>
+            <Checkbox type="checkbox" />
+            <UpdateText>
+              I want to get updated when there's an event that matches my
```
Comment (celinechoiii) [inline app/onboarding/page.tsx:50]: add "!" after interest

**Comment by celinechoiii** [comment]
YAY great job on your first PR 🎉 i put some styling / updated design related comments, but other than those, this looks amazing
